docs/src/CommutativeAlgebra/groebner_bases.md

```diff
@@ -10,9 +10,151 @@ using Oscar
 Pages = ["groebner_bases.md"]
 ```
 
-# Gröbner Bases
+# Gröbner and Standard Bases Over Fields
 
-## Computing Gröbner Bases
+Let $K[x] = K[x_1, \dots, x_n]$ be a polynomial ring over a field $K$,
+and let $>$ be a monomial ordering on $\text{Mon}_n(x)$.
+
+Given a polynomial $f\in K[x]$, write $f$ as the sum of its nonzero terms as follows:
+
+$f = a_\alpha x^\alpha + a_{\beta_1} x^{\beta_1} + \dots + a_{\beta_s} x^{\beta_s},\quad x^\alpha > x^{\beta_1} > \dots > x^{\beta_s} .$
+
+Then, with respect to $>$, we refer to $\text{LT}_>(f) := a_\alpha x^\alpha$, $\text{LM}_>(f) := x^\alpha$, $\text{LE}_>(f) := \alpha$, $\text{LC}_>(f) := a_\alpha$,
+and $\text{tail}_>(f) := f - \text{LT}_>(f)$ as the  *leading term*, the *leading monomial*, the *leading exponent*, the *leading
+coefficient*, and the *tail* of $f$, respectively.
+
+Next note that the set
+
+$U_>:= \{u\in K[x]\setminus \{0\} \mid {\text{LM}}_>(u)=1 \}$
+
+is a multiplicatively closed subset of $K[x]$. Consider the localization
+
+$K[x]_>:= K[x][U^{-1}] = \left\{ \frac{f}{u} \:\bigg|\: f \in K[x], \, u\in U_>\right\}.$
+
+Then $K[x]\subseteq K[x]_>\subseteq K[x]_{\langle x \rangle}.$ Moreover,
+
+- ``K[x] = K[x]_>`` iff $>$ is global, and
+
+- ``K[x]_> = K[x]_{\langle x \rangle}`` iff $>$ is local.
+
+Extending the notation introduced for polynomials, let now $f\in K[x]_>$. Choose $u\in U_>$ such that
+$uf\in K[x]$. Then, with respect to $>$, the  *leading term*  of $f$
+is defined to be $\text{LT}_>(f) := \text{LT}_>(uf)$ (this definition is independent of the choice of $u$).
+The *leading monomial* $\text{LM}_>(f)$, the *leading exponent* $\text{LE}_>(f)$, the
+*leading coefficient* $\text{LK}_>(f)$, and the *tail* $\text{tail}_>(f)$ of $f$ are defined similarly.
+
+!!! note
+    We use the analogous notation for elements of free modules with a given monomial ordering.
+
+!!! note
+    The OSCAR functions discussed in this section depend on a monomial ordering which may be entered as a keyword argument.
+    For a polynomial ring $R$, the respective default ordering is `degrevlex` except if $R$ is $\mathbb Z$-graded with
+	positive weights. Then the corresponding `wdegrevlex` ordering is used. For a free $R$-module $F$, the default
+	ordering is `default_ordering (R)*lex(gens(F))`.
+
+Here are some illustrating OSCAR examples:
+
+##### Examples
+
+```@repl oscar
+R, (x, y, z) = PolynomialRing(QQ, ["x", "y", "z"])
+default_ordering(R)
+F = free_module(R, 2)
+default_ordering(F)
+S, _ = grade(R, [1, 2, 3])
+default_ordering(S)
+```
+
+```@repl oscar
+R, (x, y, z) = PolynomialRing(QQ, ["x", "y", "z"])
+f = 3*z^3+2*x*y+1
+```
+
+## Division With Remainder
+
+Buchberger's algorithm for computing Gröbner bases makes use of reduction steps which rely on multivariate division with remainder.
+For the computation of standard bases, a variant of of the division algorithm due to Mora comes into play. In that context, with notation
```

Comment:
Hint: you can directly make code edit suggestions which then can be applied to the PR with one click, potentially saving everyone some work :-). See <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request> for details.

I'll do it now in this case:
```suggestion
For the computation of standard bases, a variant of the division algorithm due to Mora comes into play. In that context, with notation
```